tests/rustdoc/jump-to-def-assoc-items.rs

```diff
@@ -26,8 +26,9 @@ impl C {
     pub fn wat() {}
 }
 
-//@ has - '//a[@href="{{channel}}/core/fmt/macros/macro.Debug.html"]' 'Debug'
-//@ has - '//a[@href="{{channel}}/core/cmp/macro.PartialEq.html"]' 'PartialEq'
+// These two links must not change and in particular must contain `/derive.`!
```

Comment:
Why must they not change?  Seems like an odd thing to say without explanation considering this PR is changing them.